pkg/operator/operator.go

```diff
@@ -32,8 +36,23 @@ type Operator struct {
 }
 
 func NewOperator(ctx context.Context, operator *operator.Operator) (context.Context, *Operator) {
+	aliRegion := region.GetRegion()
```

Comment:
Let's use unified name here, `alibabaRegion`

Reply:
How about just return a error if we can't get the region info, and we will exit here